packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCommunitySelector/CWCommunitySelector.tsx

```diff
@@ -27,7 +40,7 @@ const CWCommunitySelector = ({
   return (
     <div
       className={ComponentType.CommunitySelector}
-      onClick={() => onClick(type)}
+      onClick={() => onClick({ type, chainBase })}
     >
       <div className="chain-logo-container">
         <img src={`static/img/communitySelector/${type}.svg`} alt={title} />
```

Comment:
Not a request for changes in the PR. But could we make the announcement we will move away from using cw_icons.tsx? I had not heard about this change until seeing it in this PR, so maybe it is best to get the whole team on board so we follow the same style.

Reply:
I don't think we have official statement that we won't use `cw_icons.tsx`. I just put them to static folder because I do not consider those logos as simple, single-line, vector icons. They could be 'png's as well.

Reply:
Having said that, since we use only Phosphor icons right now, we should probably not add more code into `cw_icons.tsx`.